client/src/partials/permissions/permissions.js

```diff
@@ -11,20 +6,31 @@ PermissionsController.$inject = [
   'ProjectService', 'NodeTreeService'
 ];
 
+/** 
+ * Users and Permission Controller 
+ * 
+ * This module is responsible for handling the creation
+ * of users and assigning permissions to existing modules.
+ *
+ * @todo Password insecure alert or not
+ */
 function PermissionsController($window, $translate, $http, $uibModal, util, Session, Users, Projects, NT) {
   var vm = this;
 
-  var btnTemplate =
-    '<button ng-click="grid.appScope.edit(row.entity)" style="margin: 0 5px">{{ "FORM.EDIT" | translate }}</button>' +
-    '<button ng-click="grid.appScope.editPermissions(row.entity)">{{ "PERMISSIONS.EDIT_PERMISSIONS" | translate }}</button>';
-
+  var editTemplate =
+    '<div style="margin: 0px 5px 5px 5px;"><a class="btn btn-sm btn-default btn-block" ng-click="grid.appScope.edit(row.entity)">{{ "FORM.EDIT" | translate }}</a></div>';
```

Comment:
:+1: Nice iteration.  The previous method of doing this was pretty poor.

In a subsequent pull request, do you think we should throw these in their own template files?